src/addScore.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import populate from './addRes';
+import getScore from './getScore';
+
+const URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+const id = 'f4qNRrUKtWgEVFl8Y7jY';
+const urlPost = `${URL}${id}/scores/`;
+
+export default function addPlayer() {
+  const formSubm = document.querySelector('.form');
+  const nameField = document.querySelector('.usr-name');
+  const scoreField = document.querySelector('.usr-score');
```

Comment:
- [ ] Kindly change this to es6 arrow function syntax.

Reply:
This issue is well resolved! ✔️